duckduckgo/iOS
# Drop iOS 13

Task/Issue URL: https://app.asana.com/0/414709148257752/1202397540422086/f

**Description**:
Drop support for iOS 13

**Steps to test this PR**:
General smoke tests to check if everything is OK.

Special attention to these areas:
* Onboarding
* Request review on TabViewController
* Import bookmarks
* Settings screen 

**Device Testing**:


**OS Testing**:
